File: src/main/resources/static/businessActivityCode/businessActivityCodeService.js
var services = angular.module('businessActivityCode.services',['ngResource']);

services.service('businessActivityCodeService',['$http',function($http){
		
		this.findAll = function() {
			return $http.get("/businessActivityCode");
		}		

		this.search = function(entity){
			return $http.post('/businessActivityCode/search',entity);
		}
		
		this.save = function(businessActivityCode) {
			return $http.post("/businessActivityCode",businessActivityCode);
		}
		
		this.update = function(businessActivityCode){
			return $http.put("/businessActivityCode/update", businessActivityCode);
		}
		
}]);

File: src/main/java/bank/bussinessActivityCode/BusinessActivityCodeService.java
package bank.bussinessActivityCode;

import java.util.List;

import bank.bank.Bank;

public interface BusinessActivityCodeService {

	public List<BusinessActivityCode> findAll();

	public List<BusinessActivityCode> search(BusinessActivityCode businessActivityCode);
	
	public BusinessActivityCode findOne(Long id);
	
	public BusinessActivityCode save(BusinessActivityCode businessActivityCode);
}
